sentry.go

```diff
@@ -79,14 +85,29 @@ func (so *SentryOutput) getClient(dsn string) (client *raven.Client, err error)
 
 func (so *SentryOutput) Run(or pipeline.OutputRunner, h pipeline.PluginHelper) (err error) {
 	var (
-		e      error
-		pack   *pipeline.PipelinePack
-		client *raven.Client
+		e        error
+		pack     *pipeline.PipelinePack
+		client   *raven.Client
+		contents []byte
 	)
 
+	encoder := or.Encoder()
+	if encoder == nil {
+		payloadEncoder := new(plugins.PayloadEncoder)
+		payloadEncoder.Init(new(plugins.PayloadEncoderConfig))
+		encoder = payloadEncoder
+	}
```

Comment:
My goal was to keep backwards compatibility with not specifying an encoder.  That said, it's not essential for us to keep that compatibility, and I'd be OK with breaking that to better keep with convention going forward.